addons/account/views/account_move_views.xml

```diff
@@ -525,6 +525,13 @@
                            invisible="payment_state == 'invoicing_legacy' or state != 'posted' or move_type == 'entry'"
                            optional="show"/>
                     <field name="state" widget="badge" decoration-success="state == 'posted'" decoration-info="state == 'draft'" optional="show"/>
+                    <field name="move_sent_values"
+                            string="Sent"
+                            widget="badge"
+                            decoration-success="move_sent_values == 'Sent'"
```

Comment:
and so having something like decoration-success="is_move_sent" decoration-danger="move_sent_values == 'not is_move_sent'"